### PR TITLE
Removes deprecated Tmis modules

### LINKED ISSUE #21: Remove deprecated TMIS modules

### PR DESCRIPTION
**Pull request by ee11cbb19052e40b07aac0ca060c23ee**
fixes #21

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -68,9 +68,3 @@ include ':tests:validator'
 project(':tests:validator').name = 'test-validator'
```
Comment (empbarney) [inline settings.gradle:69]: Can we remove this test-validator?

Comment (ee11cbb19052e40b07aac0ca060c23ee) [reply settings.gradle:69]: If no one is using it, I don't see why not.

Comment (empbarney) [reply settings.gradle:69]: Can you ask on the SCRUMP tomorrow?

Comment (ee11cbb19052e40b07aac0ca060c23ee) [reply settings.gradle:69]: Create separate issue for removing test-validator.

**Review by empbarney — APPROVED** [review]

**Pull request merged**